test/functional/outputs/syslog/rfc5424_test.go

```diff
@@ -23,55 +21,23 @@ var _ = Describe("[Functional][Outputs][Syslog] RFC5424 tests", func() {
 	)
 
 	BeforeEach(func() {
-		framework = functional.NewCollectorFunctionalFrameworkUsingCollector(testfw.LogCollectionType)
+		framework = functional.NewCollectorFunctionalFramework()
 		framework.MaxReadDuration = &maxReadDuration
 	})
 
 	AfterEach(func() {
 		framework.Cleanup()
 	})
-	DescribeTable("logforwarder configured with appname, msgid, and procid", func(appName, msgId, procId, expInfo string, requiresFluentd bool) {
```

Comment:
same comment here... just comment out for now and remove the "skip" statement related to fluentd and/or replace